rllib/utils/tests/test_nested_dict.py

```diff
@@ -140,6 +149,27 @@ def test_filter(self):
             dict1.filter(dict3, ignore_missing=True).asdict(), {"foo": {"a": 10}}
         )
 
+    def test_init(self):
```

Comment:
Nit: Can you also add a sub-test for empty nested_dict, i.e. NestedDict().asdict() == {}